airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -33,6 +35,17 @@
 API_VERSION = "v11"
 
 
+def change_coding(s):
+  p=re.compile(r'(?P<s>(\\\d\d\d){3,})')
+  for i in p.finditer(s):
+      old=i.group('s')
+      name=old.split('\\')
+      name=['%x' %int(g,8) for g in name if g.isdigit() ]
+      name='%'+'%'.join(name)
+      CN_name= urllib.parse.unquote(name).encode().decode('utf-8')
+      s = s.replace(old,CN_name)
+  return s.strip('"')
```

Comment:
Please add unit test for this function and add docstring explaining what is the goal for it.